README.md

```diff
@@ -232,6 +232,12 @@ If you want the docker container to run in the background pass the `-d` flag rig
 You can adjust the maximum cache size by appending `--max_size=N`, where N is
 the maximum size in Gibibytes.
 
+### Kubernetes note 
+
+Don't name your deployment `bazel-remote`!
+Kubernetes sets some env vars based on this name and they are overwritting env vars that `bazel-remote` is using.
```

Comment:
nitpick: could you use this text instead?

Kubernetes sets some environment variables based on this name, which conflict with the `BAZEL_REMOTE_*` environment variables that bazel-remote tries to parse.

Reply:
ok